wagtail/admin/views/reports/scheduled_pages.py

```diff
@@ -0,0 +1,38 @@
+from datetime import datetime
+
+import django_filters
+from django.core.exceptions import PermissionDenied
+from django.utils.translation import gettext_lazy as _
+
+from wagtail.admin.filters import DateRangePickerWidget, WagtailFilterSet
+from wagtail.admin.views.reports.base import PageReportView
+from wagtail.admin.views.scheduled_pages import get_scheduled_pages_for_user
+from wagtail.core.models import Page, UserPagePermissionsProxy
+
+
+class ScheduledPagesReportFilterSet(WagtailFilterSet):
+    go_live_at = django_filters.DateFromToRangeFilter(widget=DateRangePickerWidget)
+
+    class Meta:
+        model = Page
+        fields = ["go_live_at"]
+
+
+class ScheduledPagesView(PageReportView):
+    template_name = "wagtailadmin/reports/scheduled_pages.html"
+    title = _("Pages scheduled for publishing")
+    header_icon = "time"
+    list_export = PageReportView.list_export
```

Comment:
question: is this necessary?

Reply:
Nope, can't really see the need for exporting a list of sheduled pages.